docs/en/basic/Configuration-Settings.md

```diff
@@ -154,12 +155,12 @@ An Alluxio property can be possibly configured in multiple sources.
 In this case, its final value is determined by the following priority list, from highest priority to lowest:
 
 1. [JVM system properties (i.e., `-Dproperty=key`)](http://docs.oracle.com/javase/jndi/tutorial/beyond/env/source.html#SYS)
-2. [Environment variables](#use-environment-variables)
-3. [Property files](#use-site-property-files-recommended):
+2. [Environment variables](#environment-variables)
+3. [Property files](#alluxio-siteproperties-files-recommended):
 When an Alluxio cluster starts, each server process including master and worker searches for
 `alluxio-site.properties` within the following directories in the given order, stopping when a match is found:
 `${HOME}/.alluxio/`, `/etc/alluxio/`, and `${ALLUXIO_HOME}/conf`
```

Comment:
I think we search the current working directory as well? Isn't that why using --files and passing alluxio-site.properties in spark works for setting client properties? @calvinjia

Reply:
I think we also search on the classpath